.changeset/gold-mangos-agree.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': minor
+---
+
+Migrated the backend of the create app template to use the new backend system
+https://backstage.io/docs/backend-system/building-backends/migrating
+https://github.com/backstage/backstage/pull/21520
```

Comment:
```suggestion
---
'@backstage/create-app': patch
---

Migrated the backend of the create app template to use the new backend system. For information on how to migrate your existing backend, see the [migration guide](https://backstage.io/docs/backend-system/building-backends/migrating).
```

We don't consider template changes to be breaking changes, so moving these to `patch`. Also trimmed away the PR and issue links, which we don't include in changesets.